gr-digital/python/digital/qa_packet_headerparser_b.py

```diff
@@ -16,6 +16,16 @@
 import pmt
 
 
+def _wait_for_msgs(precondition):
+    """
+    Simple loop to check for a condition
+    """
+    stop_time = time.time() + 2.0
```

Comment:
Maybe put the default time `2.0` in a function parameter.

Reply:
It is in gr_unittest